code/game/objects/items/weapons/circuitboards/machinery/mining.dm

```diff
@@ -1,6 +1,11 @@
-/obj/item/stock_parts/circuitboard/mineral_processing
+/obj/item/stock_parts/circuitboard/mineral_processing_console
 	name = T_BOARD("mineral processing console")
-	build_path = /obj/machinery/computer/mining
+	build_path = /obj/machinery/computer/mining/processing
+	origin_tech = "{'programming':2,'engineering':2}"
+
+/obj/item/stock_parts/circuitboard/mineral_stacking_console
+	name = T_BOARD("mineral processing console")
```

Comment:
Should change the name if not using buildtype_select.

Reply:
Re: density, they are currently dense and I'm mapping them on a floor, so just going to leave them. When I redo these machines the consoles won't exist anymore.